backends-clickhouse/src/main/scala/io/glutenproject/execution/CHHashAggregateExecTransformer.scala

```diff
@@ -227,8 +227,16 @@ case class CHHashAggregateExecTransformer(
       }
     }
 
+    val aggFilterList = new util.ArrayList[ExpressionNode]()
```

Comment:
@zzcclp 
Velox backend need to pass the filter info in aggregate function from jvm to native. I also update the related code in ch backend to fix the compile errors. Can you help to review? Thanks.

Reply:
Merged into main branch firstly. If any issue, I will filed another PR to fix. Thanks.